solr/core/src/java/org/apache/solr/response/transform/CoreAugmenterFactory.java

```diff
@@ -17,9 +17,11 @@
 
 package org.apache.solr.response.transform;
 
+import jakarta.inject.Named;
 import org.apache.solr.common.params.SolrParams;
 import org.apache.solr.request.SolrQueryRequest;
 
+@Named("core")
 public class CoreAugmenterFactory extends TransformerFactory {
```

Comment:
`<RANT>` Why the hell didn't we name TransformerFactory as its rightful name, DocTransformerFactory, and name it's implementations as such as well?  I don't even need to know who/why; it's just wrong.  We have sinned and need to make amends someday.`</RANT>`